data/incipias/incipias first contact.txt

```diff
@@ -340,6 +347,7 @@ mission "Incipias: Help The Stranded 2"
 			`	You tell them to board your ship, and shortly after, you take off. Luckily, you leave just in time to avoid a powerful windstorm approaching the area.`
 				launch
 	on complete
+		log "Factions" "Incipias" `The Incipias appear to be split into two groups called the Hicemus and the Conlatio; the former occupies a space station, while the latter occupies the Incipias homeworld. The Conlatio do not appear to allow outsiders on their home planet.`
```

Comment:
Using "appear" twice is a bit stilted, although I'm not quite sure how to fix it.

Reply:
```suggestion
		log "Factions" "Incipias" `The Incipias seem to be split into two groups called the Hicemus and the Conlatio; the former occupies a space station, while the latter occupies the Incipias homeworld. The Conlatio do not appear to allow outsiders on their home planet.`
```